app/models/tasklist/TaskTitle.scala

```diff
@@ -99,4 +97,12 @@ object TaskTitle extends Enum[TaskTitle] with PlayJsonEnum[TaskTitle] {
   case object FreeRedeemableShares extends TaskTitle("FreeRedeemableSharesTitle")
 
   case object CloseCompanyLoans extends TaskTitle("CloseCompanyLoansTitle")
+
+  // Property
+  case object UkProperty extends TaskTitle("UkProperty")
+
+  case object ForeignProperty extends TaskTitle("ForeignProperty")
+
+  case object UkForeignProperty extends TaskTitle("UkForeignProperty")
```

Comment:
```suggestion
  case object UkProperty extends TaskTitle("UkPropertyTitle")

  case object ForeignProperty extends TaskTitle("ForeignPropertyTitle")

  case object UkForeignProperty extends TaskTitle("UkForeignPropertyTitle")
```

Reply:
Keys just need updating to include "Title" suffix to match the models in the submissionBE and propertyBE